scripts/_common.sh

```diff
@@ -315,3 +317,31 @@ ynh_handle_app_migration ()  {
     migration_process=1
   fi
 }
+
+
+#=================================================
+# EXPERIMENTAL HELPERS
+#=================================================
+#=================================================
+# YUNOHOST MULTIMEDIA INTEGRATION
+#=================================================
+
+# Install or update the main directory yunohost.multimedia
+#
+# usage: ynh_multimedia_build_main_dir
+ynh_multimedia_build_main_dir () {
+        wget -nv https://github.com/YunoHost-Apps/yunohost.multimedia/archive/master.zip 2>&1
+        unzip -q master.zip
```

Comment:
Could you add a check of a checksum ?

Reply:
Yes, you're right, would be better.

I'm running package check to verify my fix, and will update this PR